operators/api/v1alpha2/template_types.go

```diff
@@ -26,11 +26,24 @@ import (
 // can be instantiated in CrownLabs.
 type EnvironmentType string
 
+// +kubebuilder:validation:Enum="Standard";"Exam";"Exercise"
+
+// EnvironmentMode is an enumeration of the mode in which associated instances should be started:
+// each mode consists in presets for exposition and deployment.
+type EnvironmentMode string
+
 const (
 	// ClassContainer -> the environment is constituted by a Docker container.
 	ClassContainer EnvironmentType = "Container"
 	// ClassVM -> the environment is constituted by a Virtual Machine.
 	ClassVM EnvironmentType = "VirtualMachine"
+
+	// ModeStandard -> Normal operation (authentication, ssh, files access).
+	ModeStandard EnvironmentMode = "Standard"
+	// ModeExam -> Restricted access (no authentication, no ssh, no files access).
```

Comment:
```suggestion
	// ModeExam -> Restricted access (no authentication, no mydrive access).
```

SSH is actually still enabled, so I would omit that part of the comment.